docs/accessibility.rst

```diff
@@ -0,0 +1,182 @@
+Accessibility Testing
+======================
+
+The bok-choy framework includes the ability to test web pages for accessibility
+using `Google Accessibility Developer Tools`_.
+
+The audit rules that are checked and optionally the scope of the audit
+within the webpage itself are defined on each Page Object's definition.
+
+The general methodology for enabling accessibility auditing is:
+
+* Define the accessibility rules to check for a page
+* (optional) Define the scope for accessibility auditing of a page
+* Trigger the audits either:
+
+  * Actively, by calling the Page Object class's do_axs_audit method and then
+  asserting on the results returned.
+  * Passively, by setting the VERIFY_ACCESSIBILITY environment variable to True.
+  This will trigger an accessibility audit whenever a Page Object's wait_for_page
+  method is called, and raise an AccessibilityError if errors are found on the page.
+
+*Note that accessibility auditing is only supported with PhantomJS as the browser.*
+
+Define the accessibility rules to check for a page
+--------------------------------------------------
+
+A page object's list of accessibility rules to audit for that page are defined
+in the axs_audit_rules method. The code for the available rules can be found
+in the `audits folder`_ of the Google Accessibility Developer Tools repository.
+
+The default for page objects is to not check any rules.
+
+.. code-block:: python
+
+    def axs_audit_rules(self):
+        return None
+
+To check all rules, configure the page object's axs_audit_rules method to
+return an empty list.
+
+.. code-block:: python
+
+    def axs_audit_rules(self):
+        return []
+
+To check a single rule, or multiple rules, configure the page object's
+axs_audit_rules method to return a list with the name(s) of the rule(s).
+
+.. code-block:: python
+
+    def axs_audit_rules(self):
+        return ['badAriaAttributeValue', 'imagesWithoutAltText']
```

Comment:
The js that we're using does have a way to do this, but I only plumbed in the rules to run, not the rules to ignore. Was planning on leaving that for a future implementation. See [Audit.js](https://github.com/GoogleChrome/accessibility-developer-tools/blob/master/src/js/Audit.js#L48-60)

That said, I'm realizing it'd probably be better to rename this "audit_rules_to_run" right now in this first implementation, to leave space for adding "audit_rules_to_ignore" later.